datadog_checks_base/tests/test_db.py

```diff
@@ -1230,6 +1230,37 @@ def test_metadata(self, aggregator, datadog_agent):
         datadog_agent.assert_metadata_count(len(version_metadata))
         aggregator.assert_all_metrics_covered()
 
+    def test_hostname(self, aggregator):
+        query_manager = create_query_manager(
```

Comment:
In order to test this request change: https://github.com/DataDog/integrations-core/pull/9260/files#r622859210

You can add a query like so:

```
{
    "name": "test query",
    "query": "foo",
    "columns": [
        {'name': 'test.foo', 'type': 'count'},
        {'name': 'tag', 'type': 'tag'},
        {"name": "_source", "type": "source"}
    ],
    "extras": [
        { "name": "metric_name", "expression": "_source * 1000"}
    ]
}
```
 
And you'll get an extra metric called "metric_name", with a value being equals to 1000 times the one provided in the mock_executor

Reply:
✅